src/tunnels/views.py

```diff
@@ -1,5 +1,6 @@
 import re
 from enum import Enum
+from tkinter import SE
```

Comment:
Wow! This is my first time to meet this Class. :O

Reply:
It appears that there may be a typo with autocomplete. 🤣

Reply:
Where did it come from ??? 🤣
This is a GUI interface for Python, and it does not related to our project.